lib/utils.js

```diff
@@ -69,10 +68,30 @@ function getRandomInt(min, max) {
  * @return {function(): Node}
  */
 exports.factoryForNode = function factoryForNode(pathToTemplate) {
-  var template = fs.readFileSync(pathToTemplate)
-  var prototypeDoc = new Parser().parseFromString(template.toString())
+  const template = fs.readFileSync(pathToTemplate);
+  const prototypeDoc = new Parser().parseFromString(template.toString());
 
   return function () {
     return prototypeDoc.cloneNode(true);
   };
 };
+
+/**
+ * Standardizes PEM content to match the spec (best effort)
```

Comment:
I left it vague so that we could extend this in the future if other easily correctable errors come up. It's meant to be a catch-all reformatter for which we currently only have 1 case covered.

Reply:
I left it vague so that we could extend this in the future if other easily correctable errors come up. It's meant to be a catch-all reformatter for which we currently only have 1 case covered.